pkgs/development/python-modules/unity-test-parser/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  setuptools,
+  junit-xml,
+  approvaltests,
+  pytestCheckHook,
+}:
+buildPythonPackage rec {
+  pname = "unity-test-parser";
+  version = "0.1.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ETCLabs";
+    repo = "unity-test-parser";
+    tag = "v${version}";
+    hash = "sha256-x5kbNvQn1RxGaHGnXWKAkaBfwCpGW7lHlkHaYRGqCyE=";
+  };
+
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/ETCLabs/unity-test-parser/pull/4.patch";
```

Comment:
That URL is unstable because the PR hasn't been merged yet. Use `https://github.com/ETCLabs/unity-test-parser/commit/7f0a7bcacdffd1b89c7c87bc1be3e37482344d21` and add a comment linking to the PR.

Reply:
Done. Thank you :)